src/OneDriveMock.js

```diff
@@ -11,6 +11,7 @@
  */
 const Workbook = require('./Workbook.js');
 const StatusCodeError = require('./StatusCodeError.js');
+const { driveItemFromURL, driveItemToURL } = require('./utils.js');
```

Comment:
note, we keep the `driveItem*` methods in a separate file, so we don't need to import from `OneDrive` class to keep the mock as isolated as possible.